### DISCUSSION TITLE
lightdm greeter-hide-users=true

### DISCUSSION THREAD

**Discussion by Lichtmacher** [ltsp/ltsp#365] [Q&A]
Hi!
I wanted to hide the list of users in the lightdm.
So I created a file /etc/lightdm/lightdm.conf.d/99-xubuntu.conf
with:
```
[Seat:*]
greeter-hide-users=true
```

sudo ltsp initrd
reboot of client. It didn't work.

I logged into the client and found:
```
# Generated by `ltsp init`, see man:ltsp(8)
# You can append content here by specifying the LIGHTDM_CONF parameter
[Seat:*]
# Work around https://github.com/CanonicalLtd/lightdm/issues/49
greeter-show-manual-login=true
greeter-hide-users=false
autologin-user=
autologin-user-timeout=2
```
It seems to be hardcoded and link implies why.

My question:
how can I change it?

And by the way:
lightdm shows (whyever) the users list, but not a manual-login.

**Comment by alkisg** [comment]
> So I created a file /etc/lightdm/lightdm.conf.d/99-xubuntu.conf ... sudo ltsp initrd

The [ltsp initrd man page](https://ltsp.org/man/ltsp-initrd/) documents which folders are contained in the additional ltsp.img initrd. The /etc/lightdm folder **isn't**, so you'd need `ltsp image` for it to reach the clients.

But you don't need that, because the [ltsp.conf man page](https://ltsp.org/man/ltsp.conf/) documents how to do what you want. Just search for greeter-hide-users there.

**Comment by Lichtmacher** [comment]
Thank you for your fast respond.

"ltsp init"!!!
This program evaluates the ltsp.conf parameters?

**Comment by alkisg** [comment]
Yes, the `ltsp init` applet evaluates most, but not all, ltsp.conf parameters.

There are also ltsp.conf parameters earlier or afterwards in the client boot process, e.g. POST_INITRD_BOTTOM_x=, and there are ltsp.conf parameters for the server, for `ltsp ipxe`, for `ltsp nfs`, for `ltsp dnsmasq` etc.

**Comment by Lichtmacher** [comment]
LIGHTDM_CONF="greeter-hide-users=true"
put into the [clients] section.
sudo ltsp initrd

And it works! Thank you.
